.github/workflows/ci.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
           - 3.2
           - 3.1
           - '3.0'
+          - 2.7
```

Comment:
No need to test on Ruby 2.7 since it's already EOL. I removed that intentionally. We can allow users to use Ruby 2.7 but no interest in testing that version in CI.

```suggestion
```